TestModels/OrphanedShapes/runtimes/rust/src/extern_definitions.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::rc::Rc;
+use std::cell::UnsafeCell;
+use dafny_runtime::rcmut;
+use crate::r#_Wrappers_Compile::Result;
+
```

Comment:
If we add
`use crate::simple::orphaned::internaldafny::types`
 then we can replace `use crate::simple::orphaned::internaldafny::types` with `types` below and things will be easier to read.